antora-ui-camel/src/css/catalog.css

```diff
@@ -0,0 +1,26 @@
+.catalog-container {
+  margin-top: 20px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.catalog-entry {
+  width: 250px;
+  min-height: 140px;
+  padding: 15px;
+  margin: 5px;
```

Comment:
Try using `rem` here, I need to see the markup behind this, that seems like it could be simpler and combine `line-height` in `rem` with `min-width`. Quite possibly some of the padding/margin could come form variables in `vars.css` and be computed.